### ISSUE TITLE
Accessibility of PDF Accessibility Posters

### ISSUE THREAD

**Issue by sunnywalker** [UKHomeOffice/posters#58]
The information contained within the Accessibility posters is great! Unfortunately, the PDFs themselves are not accessible. See [WebAIM: PDF Accessibility](http://webaim.org/techniques/acrobat/) for a fair introduction to the process. (Hint: making PDFs accessible is a time consuming and frustrating experience which requires a paid version of Adobe Acrobat.)

I would love to share the PDFs as a good resource for understanding accessibility guidelines and requirements (WCAG2), but I cannot share teaching aids which themselves do not meet said guidelines.

**Comment by karypun** [comment]
@sunnywalker can you explain a bit more about where it's not accessible? Which poster or is it the whole set? We have updated the PDFs to be accessible but they may be overwritten by updates. We'll investigate this.

**Comment by karypun** [comment]
@sunnywalker the posters have been properly tagged for accessibility and we have just added a multi-page pdf version as well to enable easier download of the entire set.
